src/isa/ebpf/canonicalize_test.cc

```diff
@@ -790,14 +790,15 @@ void test3() {
 
 // expected_safe is either true for safe or false for unsafe
 void test_safety_check(inst* program, int len, bool expected_safe, string test_name) {
+  //set logger.k2functionality to optimize
```

Comment:
It would be better to explicitly set the functionality as optimize. If the default setting is changed, we don't need to adjust the test code (this is hard to detect since the tests may succeed).